BankSDK/GiniBankSDKExample/GiniBankSDKExample/Info.plist

```diff
@@ -57,7 +69,7 @@
 	<key>NSAppTransportSecurity</key>
 	<dict>
 		<key>NSAllowsArbitraryLoads</key>
-		<false/>
+		<true/>
```

Comment:
[clarification]: Do we really need to disable App Transport Security (ATS)? 
Disabling ATS means that unsecured HTTP connections are allowed.
Do we want this?

Reply:
we don't want it 🥹